pkg/lint/lintersdb/enabled_set.go

```diff
@@ -35,12 +35,9 @@ func (es EnabledSet) build(lcfg *config.Linters, enabledByDefaultLinters []*lint
 	es.debugf("Linters config: %#v", lcfg)
 	resultLintersSet := map[string]*linter.Config{}
 	switch {
-	case len(lcfg.Presets) != 0:
-		break // imply --disable-all
+	case lcfg.DisableAll, len(lcfg.Presets) != 0:
 	case lcfg.EnableAll:
 		resultLintersSet = linterConfigsToMap(es.m.GetAllSupportedLinterConfigs())
-	case lcfg.DisableAll:
-		break
```

Comment:
I don't agree with this change: the goal here is to document the possibilities not to reduce the line number.

Reply:
reverted